force-app/main/apex/examples/integration/WDdashboard.json

```diff
@@ -0,0 +1,272 @@
+{
+    "label": "Workday Reports as a Service",
```

Comment:
Cool, we should start a new directory outside of apex for sample dashboards using apex steps. Can you move this to `force-app/main/dashboards/examples`?